### PR TITLE
Used eventuate platform dependencies. #10: Used quarkus test profile for embedded database.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: customers-service/src/test/resources/application.properties [hunk 1]
```diff
@@ -1,2 +1,8 @@
 quarkus.http.test-port=9090
 eventuatelocal.kafka.bootstrap.servers=localhost:9092
+
+eventuateDatabase=h2
```
Comment (cer) [inline customers-service/src/test/resources/application.properties]: Why was this change necessary?

**Review by cer — CHANGES_REQUESTED** [review]

File: customers-service/src/test/resources/application.properties (hunk 1 above)
Comment (dartartem) [reply customers-service/src/test/resources/application.properties]: This test https://github.com/eventuate-examples/eventuate-quarkus-examples-customers-and-orders/blob/master/customers-service/src/test/java/net/chrisrichardson/eventstore/examples/customersandorders/customersservice/backend/CustomerPersistenceTest.java uses embedded database. Datasource should be configured. It transitively depends on eventuate-common-quarkus-in-memory-database where it is already defined, but properties are not applied, probably because they overwritten by properties in main/resources.

Comment (dartartem) [reply customers-service/src/test/resources/application.properties]: However, current properties define not inmemory database. I will fix.

Comment (dartartem) [reply customers-service/src/test/resources/application.properties]: fixed

Comment (cer) [reply customers-service/src/test/resources/application.properties]: > Datasource should be configured. It transitively depends on eventuate-common-quarkus-in-memory-database

But why was this change needed now? The purpose of this PR is "Used eventuate platform dependencies"

Comment (cer) [reply customers-service/src/test/resources/application.properties]: > However, current properties define not inmemory database. I will fix.

The same problem is here: https://github.com/eventuate-foundation/eventuate-common-quarkus/blob/master/eventuate-common-quarkus-in-memory-database/src/main/resources/application.properties#L5

Comment (cer) [reply customers-service/src/test/resources/application.properties]: > but properties are not applied, probably because they overwritten by properties in main/resources.

I think it's important to

(a) understand what is going on - 'probably' is vague
(b) think about possible solutions since copy/pasting properties is not that great. For example, does Quarkus provide any mechanism for having multiple property files. For example, Spring has https://www.baeldung.com/spring-test-property-source

Comment (dartartem) [reply customers-service/src/test/resources/application.properties]: > But why was this change needed now?

Without this change, build fails

>The same problem is here

I will fix

> understand what is going on - 'probably' is vague

I will investigate
